x/profile/keeper/msg_server_update_coordinator_description.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/tendermint/spn/x/profile/types"
+)
+
+func (k msgServer) UpdateCoordinatorDescription(
+	goCtx context.Context,
+	msg *types.MsgUpdateCoordinatorDescription,
+) (*types.MsgUpdateCoordinatorDescriptionResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+
+	// Check if the coordinator address is already in the store
+	coordByAddress, found := k.GetCoordinatorByAddress(ctx, msg.Address)
+	if !found {
+		return &types.MsgUpdateCoordinatorDescriptionResponse{},
+			sdkerrors.Wrap(types.ErrCoordAddressNotFound, msg.Address)
+	}
+
+	coord := k.GetCoordinator(ctx, coordByAddress.CoordinatorId)
+	if (coord == types.Coordinator{}) {
+		panic("Inconsistency error: coordinator id not exist into the keeper")
```

Comment:
```suggestion
		panic("a coordinator address is associated to a non-existent coordinator ID")
```
We don't need `Inconsistency error:` since the system panicking implies a serious issue